tests/test_globalization.py

```diff
@@ -0,0 +1,155 @@
+# coding=utf-8
+import unittest
+import socket
+from utility import random_str, decode
+from mssqltestutils import create_mssql_cli_client, shutdown
+
+
+# All tests require a live connection to test server.
+# Make modifications to mssqlutils.create_mssql_cli_client() to use a different server and database.
+# Please Note: These tests cannot be run offline.
+class GlobalizationResultSetTests(unittest.TestCase):
+    
+    def test_charset_double(self):
+        charset_list = self.get_charset_double() 
+        self.run_charset_validation(charset_list)
+    
+
+    def test_charset_four(self):
+        charset_list = self.get_charset_four() 
+        self.run_charset_validation(charset_list)
+
+
+    def test_charset_mongolian(self):
+        charset_list = self.get_charset_mongolian()
+        self.run_charset_validation(charset_list)
+
+
+    def test_charset_uyghur(self):
+        charset_list = self.get_charset_uyghur()
+        self.run_charset_validation(charset_list)
+
+
+    def test_charset_tibetian(self):
+        charset_list = self.get_charset_tibetian()
+        self.run_charset_validation(charset_list)
+
+
+    def test_charset_yi(self):
+        charset_list = self.get_charset_yi() 
+        self.run_charset_validation(charset_list)
+
+
+    def test_charset_zang(self):
+        charset_list = self.get_charset_zang() 
+        self.run_charset_validation(charset_list)
+
+    
+    def run_charset_validation(self, chars_list):
+        """
+            Verify the column names and string values in rows returned by select statement are properly encoded as unicode.
+        """
+        local_machine_name = socket.gethostname().replace('-','_').replace('.','_')
+        try:
+            client = create_mssql_cli_client()
+            max_string_length = 50
+            for chars in chars_list:
```

Comment:
>chars [](start = 16, length = 5)

nit, but important: chars and test_str are vague. Can you rename these to be more descriptive?

Reply:
synced offline. Changed to a little more descriptive words.